static-site/src/components/ListResources/types.ts

```diff
@@ -0,0 +1,45 @@
+// FIXME: is this in react-select?
+export type FilterOption = {
+    value: string
+    label: string
+}
+
+export type Group = {
+    groupName: any
+    nResources: number
+    nVersions?: number
+    lastUpdated: any
+    resources: Resource[]
+    groupUrl?: any
+    groupDisplayName?: any
+}
+
+export type Resource = {
```

Comment:
If TS were available this would have made my life a lot easier when implementing the UI. So +1 for this effort!

Reply:
types = easier to implement + easier to understand!